_benchmark/README.md

```diff
@@ -4,5 +4,8 @@ Benchmarks `build_runner` against synthetic codebases applying real generators:
 Example usage:
 
 ```
-dart run _benchmark --generator=built_value benchmark
```

Comment:
With the new one being optional, maybe show both examples?

Reply:
Done, thanks.